02-spatial-data.Rmd

```diff
@@ -169,10 +177,12 @@ text(x = ncentre_num[1], y = ncentre_num[2], labels = "Nigeria")
 ### Further work
 
 **sf** makes R data objects more closely aligned to the data model used in GDAL and GEOS, in theory making spatial data operations faster.
-The work here provides a taster of the way that **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)).
-There is a wealth of information that is available in the package's vignettes: these are highly recommended.
+The work here provides a taste of the way how **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)).
```

Comment:
"the way how" ain't correct grammar. I think it was clumsy though. My suggestion is just to use "the way":

```
The work here provides a taste of the way **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)).
```